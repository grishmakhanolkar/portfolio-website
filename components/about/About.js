import styles from './About.module.scss';
import JournalCode from '../../Images/journal-code.svg';
import Image from 'next/image';

function About() {
  return (
    <section className={styles.about}>
      <div className={styles.aboutInfo}>
        <h1 className="mx-1 mb-4 fw-bold mt-5">
          Hi, I am Grishma. Nice to meet you!
        </h1>
        <p className="fs-5 fw-light mb-5">
          I am a Web Developer. I build apps with a clean UI and a focus on the
          UX. My current tech stack is React, Next JS, Bootstrap and Material
          UI. I am open to freelancing and collaborating with others on
          interesting projects.
        </p>
      </div>
      <div className={styles.aboutCard}>
        <div className="card p-5 shadow">
          <Image
            width={50}
            height={50}
            src={JournalCode}
            alt="Frontend developer"
          />
          <div className="card-body d-flex justify-content-center flex-column p-3">
            <h3 className="p-4">Frontend Developer</h3>
            <p className="fs-5 p-2">
              I like to code things from scratch, and enjoy bringing ideas to
              life in the browser.
            </p>
            <dl>
              <dt className="p-2">Languages I speak:</dt>
              <dd className="p-2">HTML, JavaScript, CSS, NodeJS</dd>
              <dt className="p-2">Libraries I have used:</dt>
              <dd className="p-2">SCSS, ReactJS</dd>
              <dt className="p-2">Frameworks I have tried:</dt>
              <dd className="p-2">NextJS, Bootstrap, Material UI</dd>
              <dt className="p-2">Tools:</dt>
              <dd className="p-2">Github, Netlify, VSCode, Postman</dd>
            </dl>
          </div>
        </div>
      </div>
    </section>
  );
}

export default About;
