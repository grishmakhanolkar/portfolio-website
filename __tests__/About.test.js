import { render } from '@testing-library/react';
import About from '../components/about/About';
import '@testing-library/jest-dom';

it('loads and displays about', async () => {
  expect(render(<About />)).toMatchSnapshot();
});
