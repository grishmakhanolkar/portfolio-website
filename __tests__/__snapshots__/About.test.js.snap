// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loads and displays about 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="about"
      >
        <div
          class="aboutInfo"
        >
          <h1
            class="mx-1 mb-4 fw-bold mt-5"
          >
            Hi, I am Grishma. Nice to meet you!
          </h1>
          <p
            class="fs-5 fw-light mb-5"
          >
            I am a Web Developer. I build apps with a clean UI and a focus on the UX. My current tech stack is React, Next JS, Bootstrap and Material UI. I am open to freelancing and collaborating with others on interesting projects.
          </p>
        </div>
        <div
          class="aboutCard"
        >
          <div
            class="card p-5 shadow"
          >
            <span
              style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
            >
              <span
                style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2750%27%20height=%2750%27/%3e"
                  style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                />
              </span>
              <img
                alt="Frontend developer"
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
              />
              <noscript />
            </span>
            <div
              class="card-body d-flex justify-content-center flex-column p-3"
            >
              <h3
                class="p-4"
              >
                Frontend Developer
              </h3>
              <p
                class="fs-5 p-2"
              >
                I like to code things from scratch, and enjoy bringing ideas to life in the browser.
              </p>
              <dl>
                <dt
                  class="p-2"
                >
                  Languages I speak:
                </dt>
                <dd
                  class="p-2"
                >
                  HTML, JavaScript, CSS, NodeJS
                </dd>
                <dt
                  class="p-2"
                >
                  Libraries I have used:
                </dt>
                <dd
                  class="p-2"
                >
                  SCSS, ReactJS
                </dd>
                <dt
                  class="p-2"
                >
                  Frameworks I have tried:
                </dt>
                <dd
                  class="p-2"
                >
                  NextJS, Bootstrap, Material UI
                </dd>
                <dt
                  class="p-2"
                >
                  Tools:
                </dt>
                <dd
                  class="p-2"
                >
                  Github, Netlify, VSCode
                </dd>
              </dl>
            </div>
          </div>
        </div>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="about"
    >
      <div
        class="aboutInfo"
      >
        <h1
          class="mx-1 mb-4 fw-bold mt-5"
        >
          Hi, I am Grishma. Nice to meet you!
        </h1>
        <p
          class="fs-5 fw-light mb-5"
        >
          I am a Web Developer. I build apps with a clean UI and a focus on the UX. My current tech stack is React, Next JS, Bootstrap and Material UI. I am open to freelancing and collaborating with others on interesting projects.
        </p>
      </div>
      <div
        class="aboutCard"
      >
        <div
          class="card p-5 shadow"
        >
          <span
            style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
          >
            <span
              style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2750%27%20height=%2750%27/%3e"
                style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
              />
            </span>
            <img
              alt="Frontend developer"
              data-nimg="intrinsic"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
            <noscript />
          </span>
          <div
            class="card-body d-flex justify-content-center flex-column p-3"
          >
            <h3
              class="p-4"
            >
              Frontend Developer
            </h3>
            <p
              class="fs-5 p-2"
            >
              I like to code things from scratch, and enjoy bringing ideas to life in the browser.
            </p>
            <dl>
              <dt
                class="p-2"
              >
                Languages I speak:
              </dt>
              <dd
                class="p-2"
              >
                HTML, JavaScript, CSS, NodeJS
              </dd>
              <dt
                class="p-2"
              >
                Libraries I have used:
              </dt>
              <dd
                class="p-2"
              >
                SCSS, ReactJS
              </dd>
              <dt
                class="p-2"
              >
                Frameworks I have tried:
              </dt>
              <dd
                class="p-2"
              >
                NextJS, Bootstrap, Material UI
              </dd>
              <dt
                class="p-2"
              >
                Tools:
              </dt>
              <dd
                class="p-2"
              >
                Github, Netlify, VSCode
              </dd>
            </dl>
          </div>
        </div>
      </div>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
